resources/pricing.mdx

```diff
@@ -14,7 +14,8 @@ Serverless endpoint usage is billed according to the total number of tokens gene
 |--------------------|---------------------:|
 | DeepSeek-R1 MoE (671B) | $3.99           |
 | Llama-3.3 (70B)    | $0.50               |
-| Qwen2.5 (32B)*      | $0.80              | 
+| Qwen2.5 (32B)      | $0.80              | 
+| Llama-3.2-VL (11B)  | $0.15              |
```

Comment:
I think we should just add a link to the marketing website with the pricing. instead of updating in both places everytime we make changes.

Reply:
@pavel4ai